moveit_core/planning_scene/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ target_link_libraries(${MOVEIT_LIB_NAME}
   moveit_exceptions
   moveit_transforms
   moveit_collision_detection_fcl
+  moveit_collision_detection_bullet
```

Comment:
This lib shouldn't be added explicitly here as it is a plugin!

Reply:
If I remove it, the tests fail to compile as they cannot find the `collision_detection::CollisionDetectorAllocatorBullet` definition. I thought it was ok to link bullet here as there is a link for fcl already.

Reply:
Should I also add a condition here to only link the bullet plugin if the code was built with Bullet?

Reply:
Ideally, you would load the plugin via the plugin mechanism instead of instantiating it directly ;-)

Reply:
Got it! Added in c28b9ae